cmp/53fe1e2b-6a1e-4619-8da2-be8f3b9ab267/component.lp

```diff
@@ -0,0 +1,30 @@
+(librepcb_component 53fe1e2b-6a1e-4619-8da2-be8f3b9ab267
+ (name "Thermistor")
+ (description "")
+ (keywords "")
+ (author "J. R. Schmid")
+ (version "0.1")
+ (created 2020-05-10T13:35:28Z)
+ (deprecated false)
+ (category 1039f038-20a6-4bfe-89c1-99f34fbb45bd)
+ (schematic_only false)
+ (default_value "{{PARTNUMBER or DEVICE}}")
```

Comment:
The value can default to `{{RESISTANCE}}`, see other comment about the value.
```suggestion
 (default_value "{{RESISTANCE}}")
```

Reply:
Same question as above: are we talking about table "Symbol Variants" or table "Attributes"?

Reply:
This is the default value of the component (on the right side, below the category). If you click on "Commit suggestion" and do a git pull, you should end up with the fixed component :slightly_smiling_face: